.rubocop.yml

```diff
@@ -246,6 +249,8 @@ Style/GlobalStdStream: # (new in 0.89)
   Enabled: true
 Style/HashAsLastArrayItem: # (new in 0.88)
   Enabled: true
+Style/HashConversion: # (new in 1.10)
+  Enabled: true
```

Comment:
Rubocop changed while I was working on this PR, so I included the new rules here.